pkg/core/transaction/transaction.go

```diff
@@ -449,3 +449,9 @@ func (t *Transaction) HasSigner(hash util.Uint160) bool {
 	}
 	return false
 }
+
+// InvalidateCache invalidates transaction hash and size cache.
+func (t *Transaction) InvalidateCache() {
```

Comment:
Wow, that's a substantial change, because it's a whole new exported API. And I'm not so sure about it.